lib/main_common.pm

```diff
@@ -1402,7 +1402,7 @@ sub load_extra_tests_textmode {
     # Currently for our SLE12 validation tests we are not using a
     # registered SLE installation so we should not schedule the test
     # modules.
-    load_docker_tests if (check_var('ARCH', 'x86_64') && ((is_sle('12-SP2+') && (is_sle('<12-SP4') || is_sle('15+')) || !is_sle)));
+    load_docker_tests if (check_var('ARCH', 'x86_64') && (is_sle('12-SP2+') || is_sle('15+') || !is_sle));
```

Comment:
'12-SP+' should include '15+' ,right ?

Reply:
yes

Reply:
It looks we were working on similar thing https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4810  my goal is to disable docker tests on SLE12-SP2.

Reply:
Then why not simply ?
```bash
load_docker_tests if (check_var('ARCH', 'x86_64') && (is_sle('12-SP3+') || !is_sle));
```

Reply:
Sounds good. Should i cancel my PR and wait for this one?

Reply:
@czerw, if you fix your PR quickest than @okurz, I will merge yours first, so you don't have to wait for #4805.

Reply:
@okurz, please be aware that I merged @czerw's PR #4810

Reply:
ok. I assume this broke SLE12SP4 tests because without *this* PR the docker test module will fail. So I would like to rebase and then merge. And we can discuss which repo to take or to register later.